<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>911</width>
    <height>638</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>*NAME*</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>911</width>
      <height>571</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Журнал</string>
     </attribute>
     <widget class="QPushButton" name="add_expense_btn">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>470</y>
        <width>150</width>
        <height>23</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Добавить</string>
      </property>
     </widget>
     <widget class="QPushButton" name="add_gain_btn">
      <property name="geometry">
       <rect>
        <x>460</x>
        <y>470</y>
        <width>150</width>
        <height>23</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Добавить</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="total">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>510</y>
        <width>821</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>210</y>
        <width>71</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Расходы</string>
      </property>
     </widget>
     <widget class="QPushButton" name="save_btn">
      <property name="geometry">
       <rect>
        <x>660</x>
        <y>120</y>
        <width>141</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Сохранить</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>660</x>
        <y>210</y>
        <width>71</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Доходы</string>
      </property>
     </widget>
     <widget class="QTableWidget" name="gains_table">
      <property name="geometry">
       <rect>
        <x>460</x>
        <y>250</y>
        <width>430</width>
        <height>211</height>
       </rect>
      </property>
      <column>
       <property name="text">
        <string>Имя</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Категория</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Сумма</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Примечание</string>
       </property>
      </column>
     </widget>
     <widget class="QPushButton" name="del_gain_btn">
      <property name="geometry">
       <rect>
        <x>740</x>
        <y>470</y>
        <width>150</width>
        <height>23</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Удалить</string>
      </property>
     </widget>
     <widget class="QTableWidget" name="expenses_table">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>250</y>
        <width>430</width>
        <height>211</height>
       </rect>
      </property>
      <column>
       <property name="text">
        <string>Имя</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Категория</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Сумма</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Примечание</string>
       </property>
      </column>
     </widget>
     <widget class="QPushButton" name="load_btn">
      <property name="geometry">
       <rect>
        <x>660</x>
        <y>50</y>
        <width>141</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Загрузить</string>
      </property>
     </widget>
     <widget class="QPushButton" name="del_expense_btn">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>470</y>
        <width>150</width>
        <height>23</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Удалить</string>
      </property>
     </widget>
     <widget class="QCalendarWidget" name="calendar">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>601</width>
        <height>171</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>9</pointsize>
       </font>
      </property>
     </widget>
     <widget class="Line" name="line">
      <property name="geometry">
       <rect>
        <x>440</x>
        <y>210</y>
        <width>20</width>
        <height>281</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>510</y>
        <width>61</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Итого:</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Регулярные платежи</string>
     </attribute>
     <widget class="QPushButton" name="del_spending_btn">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>270</y>
        <width>150</width>
        <height>23</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Удалить</string>
      </property>
     </widget>
     <widget class="QPushButton" name="add_spending_btn">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>270</y>
        <width>150</width>
        <height>23</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Добавить</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="reg_total">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>310</y>
        <width>821</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>660</x>
        <y>10</y>
        <width>71</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Доходы</string>
      </property>
     </widget>
     <widget class="QPushButton" name="del_income_btn">
      <property name="geometry">
       <rect>
        <x>740</x>
        <y>270</y>
        <width>150</width>
        <height>23</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Удалить</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>310</y>
        <width>61</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Итого :</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>10</y>
        <width>71</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Расходы</string>
      </property>
     </widget>
     <widget class="QTableWidget" name="income_table">
      <property name="geometry">
       <rect>
        <x>460</x>
        <y>50</y>
        <width>430</width>
        <height>211</height>
       </rect>
      </property>
      <column>
       <property name="text">
        <string>Имя</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Категория</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Сумма</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Период</string>
       </property>
      </column>
     </widget>
     <widget class="Line" name="line_3">
      <property name="geometry">
       <rect>
        <x>440</x>
        <y>10</y>
        <width>20</width>
        <height>281</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
     <widget class="QPushButton" name="add_income_btn">
      <property name="geometry">
       <rect>
        <x>460</x>
        <y>270</y>
        <width>150</width>
        <height>23</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Добавить</string>
      </property>
     </widget>
     <widget class="QTableWidget" name="spending_table">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>430</width>
        <height>211</height>
       </rect>
      </property>
      <column>
       <property name="text">
        <string>Имя</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Категория</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Сумма</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Период</string>
       </property>
      </column>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>911</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
